services/data/src/links/RestAPILink/queryToRequestOptions/multipartFormDataMatchers.ts

```diff
@@ -0,0 +1,22 @@
+import { ResolvedResourceQuery, FetchType } from '../../../engine'
+
+/*
+ * Requests that expect a "multipart/form-data" Content-Type have been collected by scanning
+ * the developer documentation:
+ * https://docs.dhis2.org/master/en/developer/html/dhis2_developer_manual_full.html
+ */
+
+// POST to 'fileResources' (upload a file resource)
+export const fileResourceUpload = (
+    type: FetchType,
+    { resource }: ResolvedResourceQuery
+) => type === 'create' && resource === 'fileResources'
+
+// POST to `staticContent/${key}` (upload staticContent: logo_banner | logo_front)
+export const isStaticContentUpload = (
+    type: FetchType,
+    { resource }: ResolvedResourceQuery
+) => {
+    const pattern = /^staticContent\/(?:logo_banner|logo_front)$/
+    return type === 'create' && pattern.test(resource)
+}
```

Comment:
There are definitely going to be a few others here - that I know of **POST /api/apps** and metadata import (both accept ZIP files) but I think this is fine for now and we can expand as we run into more cases!!

Reply:
Thanks for pointing that out. I also seemed to have missed `messageConversations/attachments`. In 28e97d6 I added that one and also `apps` as per your suggestion. However, I've skipped the metadata import endpoint because that seems to accept all sorts of things... Let's tackle that when it it pops up.